app/controllers/admin/pending_orders_controller.rb

```diff
@@ -0,0 +1,5 @@
+class Admin::PendingOrdersController < ApplicationController
+  def index
+    @pending_orders = Order.includes(:user).where(status: 'Pending')
```

Comment:
lets use small letters for statuses

`@pending_orders = Order.includes(:user).where(status: 'pending')`